docker/Dockerfile

```diff
@@ -83,11 +76,20 @@ RUN wget -r --no-parent -nH --cut-dirs=2 http://www.abisource.com/downloads/link
 # Unpack the sources, too.
 RUN tar -zxf current/link-grammar-5*.tar.gz
 
-RUN (cd link-grammar-5.*/; ./configure; make -j6; sudo make install; ldconfig)
+RUN (cd link-grammar-5.*/; ./configure; make -j6; make install; ldconfig)
+
+RUN LINKGRAMMAR_JAR=`find ./link-grammar* -name linkgrammar*.jar` &&\
+    LINKGRAMMAR_VERSION=`echo $LINKGRAMMAR_JAR | grep -oP '(?<=-)\d+\.\d+\.\d+(?=\.)'` &&\
+    mvn install:install-file \
+    -Dfile=$LINKGRAMMAR_JAR \
+    -DgroupId=org.opencog \
+    -DartifactId=linkgrammar \
+    -Dversion=$LINKGRAMMAR_VERSION \
+    -Dpackaging=jar
 
```

Comment:
the correct solution would be to move this to the link-grammar automake file, so that if mvn is found, then `mvn install` is called.

Reply:
Agree

Reply:
See the proposed fix for link-grammar and its drawbacks in https://github.com/opencog/link-grammar/pull/929